cdc/puller/ddl_puller_test.go

```diff
@@ -647,15 +657,22 @@ func TestResolvedTsStuck(t *testing.T) {
 	mockPuller := newMockPuller(t, startTs)
 	ctx := cdcContext.NewBackendContext4Test(true)
 	up := upstream.NewUpstream4Test(nil)
+	f, err := filter.NewFilter(config.GetDefaultReplicaConfig(), "")
+	require.Nil(t, err)
 	schemaStorage, err := entry.NewSchemaStorage(nil,
 		startTs,
-		true,
+		ctx.ChangefeedVars().Info.Config.ForceReplicate,
```

Comment:
Why we change `true` to `Config.ForceReplicate`? It was a bug?

Reply:
There was no bug; I just tried to keep it the same with line 671. There is a changefeedCtx that contains the configuration of a changefeed, so using the changefeed's configuration instead of a magic true seems more reasonable to me.